### PR TITLE
variable,printer: add a session variable to show configuration.

### PR DESCRIPTION
**Pull request by coocood**
And log the config at server start.

### CODE REVIEW

File: util/printer/printer.go [hunk 1]
```diff
@@ -37,6 +39,11 @@ func PrintTiDBInfo() {
 	log.Infof("Git Branch: %s", TiDBGitBranch)
 	log.Infof("UTC Build Time:  %s", TiDBBuildTS)
 	log.Infof("GoVersion:  %s", GoVersion)
+	configJSON, err := json.Marshal(config.GetGlobalConfig())
+	if err != nil {
+		panic(err)
+	}
+	log.Infof("Config: %s", configJSON)
```
Comment (shenli) [inline util/printer/printer.go]: How about the readability of this?

Comment (coocood) [reply util/printer/printer.go]: I think makes it one line is easier for parsing by programs.

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**